components/gc-minister/index.json-ld

```diff
@@ -185,6 +250,25 @@
 					"code": "<section class=\"gc-minister\">\n\t<h2>Minister</h2>\n\t<div class=\"row wb-eqht-grd\">\n\t\t<div class=\"col-xs-12 col-md-6\">\n\t\t\t<h3><a href=\"#\">The Honourable [Minister 3 name]</a></h3>\n\t\t\t<p>[Official title]</p>\n\t\t\t<ul>\n\t\t\t\t<li><a href=\"\">Mandate letter</a></li>\n\t\t\t\t<li><a href=\"\">Briefing book</a></li>\n\t\t\t</ul>\n\t\t</div>\n\t</div>\n</section>"
 				}
 			]
+		},
+		{
+			"@id": "_:iteration_minister_2",
+			"name": "Minister - Iteration 2",
+			"date": "2025-04",
+			"detectableBy": ".gc-minister",
```

Comment:
Add the changes that this iteration brings. Either under "breaking", "additions", or "fixes". I will let you evaluate in which category these new changes fall under.

Reply:
Fixed in commit " Updated documentation"